assets/stylesheets/foreground.css

```diff
@@ -713,13 +714,13 @@ div.mw-htmlform-field-HTMLCheckField div.mw-label { height:0; }
 
 
 /* Jamie's Header Hider */
```

Comment:
Yes. We should probably open an issue to discuss this. If we do decide to let the user decide we should add a notice in read me file on how to hide the main page title. But this cuss only applies to English language wikis since other wikis with different languages won't have this main page title.